tensorflow/core/api_def/base_api/api_def_Asinh.pbtxt

```diff
@@ -1,4 +1,13 @@
 op {
   graph_op_name: "Asinh"
   summary: "Computes inverse hyperbolic sine of x element-wise."
+  description: <<END
+  Given an input tensor, this function computes inverse hyperbolic sine
+  for every element in the tensor. Both input and output has a range of
+  (-infinity, infinity).
+  ```python
+  x = tf.constant([-2, -0.5, 1, 1.2, 200, 10000])
+  tf.math.asinh(x, name ='asinh') ==> [-1.4436355 -0.4812118  0.8813736  1.0159732  5.991471   9.903487 ]
```

Comment:
I'd remove the name arg, it's just a distraction. There's a spurious space before `]`.